2025-02-23 14:32:46.113089-05:00 database: close
org.h2.message.DbException: IO Exception: "Closing" [90028-232]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.close(Store.java:374)
	at org.h2.engine.Database.closeOpenFilesAndUnlock(Database.java:1292)
	at org.h2.engine.Database.closeImpl(Database.java:1254)
	at org.h2.engine.Database.close(Database.java:1173)
	at org.h2.engine.Database.onShutdown(Database.java:1158)
	at org.h2.engine.OnExitDatabaseCloser.onShutdown(OnExitDatabaseCloser.java:85)
	at org.h2.engine.OnExitDatabaseCloser.run(OnExitDatabaseCloser.java:114)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "Closing" [90028-232]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 8 more
Caused by: org.h2.mvstore.MVStoreException: java.lang.AssertionError [2.3.232/3]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.MVStore.executeFilestoreOperation(MVStore.java:926)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:463)
	at org.h2.mvstore.RandomAccessStore.compactStore(RandomAccessStore.java:443)
	at org.h2.mvstore.FileStore.compactStore(FileStore.java:890)
	at org.h2.mvstore.FileStore.stop(FileStore.java:275)
	at org.h2.mvstore.MVStore.closeStore(MVStore.java:693)
	at org.h2.mvstore.MVStore.close(MVStore.java:643)
	at org.h2.mvstore.db.Store.close(Store.java:364)
	... 6 more
Caused by: java.lang.AssertionError
	at org.h2.mvstore.RandomAccessStore.moveChunkInside(RandomAccessStore.java:630)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:569)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:481)
	at org.h2.mvstore.RandomAccessStore.lambda$compactMoveChunks$1(RandomAccessStore.java:468)
	at org.h2.mvstore.FileStore.executeFileStoreOperation(FileStore.java:1893)
	at org.h2.mvstore.MVStore.executeFilestoreOperation(MVStore.java:922)
	... 13 more
2025-02-23 19:49:52.898518-05:00 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: java.lang.AssertionError: 45056 != 40960 [2.3.232/3]" [50000-232]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1546)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1533)
	at org.h2.mvstore.FileStore.lambda$serializeAndStore$3(FileStore.java:1436)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: java.lang.AssertionError: 45056 != 40960 [2.3.232/3]" [50000-232]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 12 more
Caused by: org.h2.mvstore.MVStoreException: java.lang.AssertionError: 45056 != 40960 [2.3.232/3]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	... 7 more
Caused by: java.lang.AssertionError: 45056 != 40960
	at org.h2.mvstore.RandomAccessStore.shrinkStoreIfPossible(RandomAccessStore.java:679)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:377)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:28)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1528)
	... 6 more
2025-02-23 19:49:52.931297-05:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: The database has been closed [90098-232]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.engine.SessionLocal.getTransaction(SessionLocal.java:1616)
	at org.h2.engine.SessionLocal.startStatementWithinTransaction(SessionLocal.java:1637)
	at org.h2.command.Command.executeQuery(Command.java:190)
	at org.h2.jdbc.JdbcConnection.isReadOnly(JdbcConnection.java:586)
	at com.zaxxer.hikari.pool.PoolBase.setupConnection(PoolBase.java:404)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:365)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:202)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:461)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:724)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:703)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
Caused by: org.h2.mvstore.MVStoreException: java.lang.AssertionError: 45056 != 40960 [2.3.232/3]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1533)
	at org.h2.mvstore.FileStore.lambda$serializeAndStore$3(FileStore.java:1436)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	... 4 more
Caused by: java.lang.AssertionError: 45056 != 40960
	at org.h2.mvstore.RandomAccessStore.shrinkStoreIfPossible(RandomAccessStore.java:679)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:377)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:28)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1528)
	... 6 more
2025-02-23 19:54:06.486630-05:00 database: close
org.h2.message.DbException: IO Exception: "Closing" [90028-232]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.close(Store.java:374)
	at org.h2.engine.Database.closeOpenFilesAndUnlock(Database.java:1292)
	at org.h2.engine.Database.closeImpl(Database.java:1254)
	at org.h2.engine.Database.close(Database.java:1173)
	at org.h2.engine.Database.onShutdown(Database.java:1158)
	at org.h2.engine.OnExitDatabaseCloser.onShutdown(OnExitDatabaseCloser.java:85)
	at org.h2.engine.OnExitDatabaseCloser.run(OnExitDatabaseCloser.java:114)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "Closing" [90028-232]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 8 more
Caused by: org.h2.mvstore.MVStoreException: java.lang.AssertionError [2.3.232/3]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.MVStore.executeFilestoreOperation(MVStore.java:926)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:463)
	at org.h2.mvstore.RandomAccessStore.compactStore(RandomAccessStore.java:443)
	at org.h2.mvstore.FileStore.compactStore(FileStore.java:890)
	at org.h2.mvstore.FileStore.stop(FileStore.java:275)
	at org.h2.mvstore.MVStore.closeStore(MVStore.java:693)
	at org.h2.mvstore.MVStore.close(MVStore.java:643)
	at org.h2.mvstore.db.Store.close(Store.java:364)
	... 6 more
Caused by: java.lang.AssertionError
	at org.h2.mvstore.RandomAccessStore.moveChunkInside(RandomAccessStore.java:630)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:569)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:481)
	at org.h2.mvstore.RandomAccessStore.lambda$compactMoveChunks$1(RandomAccessStore.java:468)
	at org.h2.mvstore.FileStore.executeFileStoreOperation(FileStore.java:1893)
	at org.h2.mvstore.MVStore.executeFilestoreOperation(MVStore.java:922)
	... 13 more
